projects/gen-ai.md

```diff
@@ -0,0 +1,172 @@
+
+[OpenLLMetry](https://github.com/traceloop/openllmetry) project was created with the intention [to create space under CNCF](https://github.com/cncf/sandbox/issues/67) and gave a start to OTel [LLM Semantic Convention](https://github.com/open-telemetry/community/blob/main/projects/llm-semconv.md) project.
+
+GenAI Instrumentation libraries like OpenLLMetry, OpenLIT, LangTrace and other provides general-purpose OTel instrumentations for GenAI clients, frameworks and related components.
+
+By providing GenAI community with the space under OpenTelemetry we will be able to start hosting vendor-agnostic instrumentation libraries (such as WIP one [OpenAI Python](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/2759)) and [OpenAI JS](https://github.com/open-telemetry/opentelemetry-js-contrib/pull/2402) and eventually we will be able to move instrumentations from current GenAI Observability tools to OTel (or to upstream libraries).
+
+It helps GenAI observability community to
+
+- find solutions for new observability problems such as verbose events or correlation with async response evaluations
+- discuss implementation challenges
+- coordinate and prioritize common efforts
+
+### Goals, objectives, and requirements
+
+GenAI observability project is a new OTel cross-cutting SIG that intends to provide the GenAI community with a space to evolve and maintain GenAI related instrumentations in a **vendor-agnostic way**.
+
+It includes:
+
+- LLM Semantic Conventions
+- Instrumentations for GenAI client libraries, frameworks, or other components.
+
+The eventual goal is to contribute instrumentation libraries to upstream components or to otel-contrib repos when upstream contributions are not possible.
+
+During initial development and experimentation, we may need to release libraries more frequently than it's possible for contrib and then may temporarily host instrumentation libraries in a `opentelemetry-{language}-genai` repository(ies).
+
+This would help GenAI observability vendors that will depend on these libraries to patch issues and adapt to external changes.
+
+## Deliverables
+
+Immediate term:
+- Ship OTel instrumentation libraries for OpenAI (or any other GenAI client) in Python and JS following existing conventions
+
+Middle term:
+- Ship OpenTelemetry (or native) instrumentations for popular GenAI client libraries in Python and JS covering chat calls
+- Evolve GenAI semantic conventions to cover other popular GenAI operations such as embeddings, image or audio generation
+
+As a result we should have feature parity with the instrumentations of existing GenAI Observability vendors for a set of client
+instrumentation libraries that all vendors can depend upon.
+
+Long term:
+- Implement instrumentations for GenAI orchestrators and LLM frameworks for popular libraries in different languages
```

Comment:
```suggestion
- Implement instrumentations for GenAI orchestrators and LLM/SLM frameworks for popular libraries in different languages
```

Or just say GenAI frameworks. There is going to be an increase of systems that include different categories of hosted LLMs and local SLMs that work together in a solution.